lighthouse-cli/chrome-launcher.ts

```diff
@@ -70,10 +70,16 @@ class ChromeLauncher {
   flags() {
     const flags = [
       `--remote-debugging-port=${this.port}`,
-      '--disable-extensions',
       '--disable-translate',
-      '--disable-default-apps',
-      '--no-first-run',
+      '--disable-extensions', // disable all chrome extensions entirely
+      // '--ignore-certificate-errors', // Ignore certificate errors, like self-signed
```

Comment:
@wardpeet mentioned maybe exposing this as a CLI flag. Talking about it offline with @brendankenny, we could do a `--chrome-flags="--ignore-certificate-errors"` kinda thing. That's not bad at all.